aws-observability/json/Alerts-App.json

```diff
@@ -977,7 +977,7 @@
     },
     {
       "name": "AWS DynamoDB - Multiple Tables deleted",
-      "description": "This alert fires when we detect multiple failed operations for Elasticache service within 15 minutes",
+      "description": "This alert fires when five or more tables are deleted within 10 minutes.",
```

Comment:
Monitor timerange is configured for 15 min instead of 10 min. Accordingly make desc for 15 min.